src/core/thread/mle_router.cpp

```diff
@@ -3334,7 +3334,7 @@ ThreadError MleRouter::RestoreChildren(void)
     for (uint8_t i = 0; ; i++)
     {
         Child *child;
-        otChildInfo childInfo;
+        ChildInfo childInfo;
         uint16_t length;
 
         length = sizeof(childInfo);
```

Comment:
Thanks for pointing this out.  That behavior is actually inconsistent with the [documentation](https://github.com/openthread/openthread/blob/master/include/openthread/platform/settings.h#L150).  The documented behavior seems better to me and we should implement to that.